### DISCUSSION TITLE
Displaying collection path to Vault items

### DISCUSSION THREAD

**Discussion by Satrier** [dani-garcia/vaultwarden#3286] [Ideas]
Hi!

- [x] There are not enough custom icons, for example, SSH, RDP, etc.

And also in All vaults All items, I don’t understand which collection which item is located in.

**Comment by Satrier** [comment]
>There are not enough custom icons, for example, SSH, RDP, etc.
Its done - added icons to /./icon_cache

**Comment by BlackDex** [comment]
I'm not sure what you mean here. It looks to me like some UI stuff. If that is the case, we can't help you there.
This is all Bitwarden handled, and not Vaultwarden.
